unabridged-works-volume-1/original/05-babylon-the-great.adoc

```diff
@@ -2105,21 +2198,27 @@ Though thou art grown much higher, wiser, and greater otherwise,
 yet there thou hast lost: and happy wilt thou be,
 if all thy gains ever since may become loss to thee for the recovering of this.
 
-=== SOME CONSIDERATIONS
+=== Some Considerations
 
+[.section-summary-preface]
 Helping out of the Suburbs or Out-skirts of Babylon,
```

Comment:
While you're in there, would you mind changing this to `Outskirts` instead of `Outskirts`?